nemo_skills/pipeline/eval.py

```diff
@@ -197,12 +211,25 @@ def eval(
         pass
 
     cluster_config = get_cluster_config(cluster, config_dir)
-    check_if_mounted(cluster_config, output_dir)
+    cluster_config = resolve_mount_paths(cluster_config, mount_paths)
```

Comment:
Feels like we can have a utility that does all of this and takes all boolean arguments as well as all files to check and log_dir and then just call it as one line in all pipelines to avoid duplication